plugins/inventory/openshift.py

```diff
@@ -22,7 +22,7 @@
       plugin:
         description: token that ensures this is a source file for the 'openshift' plugin.
         required: True
-        choices: ['openshift']
+        choices: ['openshift', 'redhat.openshift.openshift']
```

Comment:
Shouldn't this be `community.okd.openshift`?

Reply:
Yes, you are right.